fixity/fixity.py

```diff
@@ -305,7 +309,9 @@ def get_handler(stream, timestamps, log_level=None):
     return handler
 
 
-def main(argv=None, logger=None, stream=None):
+def main(
+    argv: List[str] = None, logger: logging.Logger = None, stream: TextIO = None
+) -> Union[int, bool]:
```

Comment:
Have you seen the `try... except` blocks inside this function? Don't they also return exceptions?